src/main/java/org/biouno/unochoice/AbstractUnoChoiceParameter.java

```diff
@@ -89,6 +93,7 @@
     protected AbstractUnoChoiceParameter(String name, String description) {
         super(name, description);
         randomName = Utils.createRandomParameterName("choice-parameter", "");
+        _uuid = UUID.randomUUID();
```

Comment:
Interesting, we inherited this bug from the dynamic-parameter. I believe @imoutsatsos told me about it some time ago. IIRC, he was having problems backing up configuration (I think there were changes in these random names, regardless of whether it made sense or not).

See https://issues.jenkins-ci.org/browse/JENKINS-32149 for the fix.